app/(auth)/change-password/page.tsx

```diff
@@ -63,27 +65,53 @@ export default function ChangePassword() {
   )
 
   return (
-    <>
-      <Heading>Change Password</Heading>
-      <Box as="form" onSubmit={changePasswordHandler}>
-        <Flex flexDirection={'column'}>
+    <Box
+      as="main"
+      w="100vw"
+      minH={{ base: 'calc(100vh - 145px)', md: 'calc(100vh - 210px)' }}
+      color={color}
+    >
+      <VStack
+        mt={{ base: '20px', md: '40px' }}
+        mx="auto"
+        w={{ base: '380px', md: '480px' }}
+        px={{ base: '16px', md: '80px' }}
+      >
+        <Heading
+          as="h1"
+          fontSize={{ base: '2xl', md: '4xl' }}
+          mb="40px"
+          w="100%"
+          textAlign={{ base: 'left', md: 'center' }}
+        >
+          Change Password
+        </Heading>
+        <VStack
+          spacing={{ base: '30px', md: '40px' }}
+          as="form"
+          onSubmit={changePasswordHandler}
+        >
           <FormControl isInvalid={!!errors.oldPassword}>
             <FormLabel>Old Password</FormLabel>
-            <Input type="password" {...register('oldPassword')} />
+            <InputForm
+              {...register('oldPassword')}
+              w={{ base: '358px', md: '480px' }}
+              showInput={true}
```

Comment:
Maybe it does make sense if you pass the props as "hasEyeIcon". `showInput={true}` sounds like you always show input, but you hide input after a user click the icon.
You can use the same password input in login form too!

Reply:
That makes total sense! I changed the name!